lib/main.rb

```diff
@@ -1,10 +1,8 @@
-require_relative './helper'
 require_relative './fibonacci'
```

Comment:
I understand that the main file is for you to test / demo it. It's a useful addition for this purpose, but I'd be totally fine with a demo method that you had before (`fibonacci.run(12)`).